spec/services/create_refund_spec.rb

```diff
@@ -68,6 +68,20 @@
         end
       end
 
+      context "when there isn't a comment" do
+        let(:attributes) do
+          {
+            value: 100.10,
+            financial_quarter: 1,
+            financial_year: 2020
+          }
+        end
+
+        it "has an error" do
+          expect(subject.object.errors.messages.values.to_s).to include("Enter a comment describing the need for the refund")
+        end
+      end
+
```

Comment:
I guess we want to squash this commit into the previous WIP?

Reply:
Confused as to which commits you'd like squashing, but very happy to tidy up the commits once we're ready to go. (I've broadly left them as-is because losing the context for the changes is unhelpful right now, but yes, good to have tidy commits in future.)

Reply:
You've already squashed them! Resolving :)